Rakefile

```diff
@@ -538,7 +538,28 @@ end
 
 installed_pod_bin = File.join(BUNDLE_DESTROOT, 'bin/pod')
 file installed_pod_bin => rubygems_update_dir do
-  install_gem 'cocoapods', install_cocoapods_version
+  execute "bundling", ["rm", "-rf", WORKBENCH_DIR + "/ruby"]
+  # Can figure this one out later
+  bundler = "/Users/orta/.rvm/gems/ruby-2.1.3/wrappers/bundle"
```

Comment:
I really couldn't figure out how to use the user's install of bundler, tried a bunch of things